articles/install-guide/index.md

```diff
@@ -17,237 +17,11 @@ Learn how to install the Microsoft Quantum Development Kit (QDK), so that you ca
-
-1. Pre-requisites
-
-   - [VS Code](https://code.visualstudio.com/download)
-   - [.NET Core SDK 3.0 or later](https://www.microsoft.com/net/download)
-
-1. Install the Quantum VS Code extension
-
-    - Install the [VS Code extension](https://marketplace.visualstudio.com/items?itemName=quantum.quantum-devkit-vscode)
-
-1. Install the Quantum project templates:
-
-   - Go to **View** -> **Command Palette**
-   - Select **Q#: Install project templates**
-
-    You now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
-
-1. Verify the installation by creating a `Hello World` application
-
-    - Create a new project:
-
-        - Go to **View** -> **Command Palette**
-        - Select **Q#: Create New Project**
-        - Navigate to the location on the file system where you would like to create the application
-        - Click on the **Open new project...** button, once the project has been created
-
-    - Run the application:
-
-        - Go to **Debug** -> **Start Without Debugging**
-        - You should see the following text in the output window `Hello quantum world!`
-
-> [!NOTE]
-> * > * Workspaces with multiple root folders are not currently supported by the Visual Studio Code extension. If you have multiple projects within one VS Code workspace, all projects need to be contained within the same root folder.
-
-## Develop with C#, using the `dotnet` command-line tool
-
-Of course, you can also build and run Q# programs from the command line by simply installing the .NET Core SDK and the QDK project templates. 
-
-1. Pre-requisites
-
-    - [.NET Core SDK 3.0 or later](https://www.microsoft.com/net/download)
-
-1. Install the Quantum project templates for .NET
-
-    ```bash
-    dotnet new -i Microsoft.Quantum.ProjectTemplates
-    ```
-
-    You now have the Quantum Development Kit installed and ready to use in your own applications and libraries.
-
-1. Verify the installation by creating a `Hello World` application
-
-    - Create a new application
-
-       ```bash
-       dotnet new console -lang Q# -o runSayHello
-       ```
-
-    - Navigate to the new application directory
-
-       ```bash
-       cd runSayHello
-       ```
-
-    You should see that two files have been created, along with the project files of the application: a Q# file (`Operation.qs`) and a C# host file (`Driver.cs`).
-
-    - Run the application
-
-        ```bash
-        dotnet run
-        ```
-
-        You should see the following output: `Hello quantum world!`
-
-## What's next?
-
-Now that you have installed the Quantum Development Kit in your preferred environment, you can write and run [your first quantum program](xref:microsoft.quantum.write-program).
+- [Q# + Python:](xref:microsoft.quantum.install.python) choose this environment if you want to combine Python and Q# to create a Python host program that calls Q# operations whenever they are needed.
```

Comment:
```suggestion
- [Install Q# for Python:](xref:microsoft.quantum.install.python) choose this environment if you want to combine Python and Q# to create a Python host program that calls Q# operations whenever they are needed.
```

Reply:
Since here we are listing the different environments I prefer to prescind of the "Install..." part since it's already understood by the context (we are in the installation overview page).

Reply:
Having "install" is good as it helps to clarify the intent, and will perform better in search